x-pack/plugins/ml/common/util/job_utils.ts

```diff
@@ -223,7 +235,7 @@ export function getSingleMetricViewerJobErrorMessage(job: CombinedJob): string |
 
   if (isChartableTimeSeriesViewJob === false) {
     return i18n.translate('xpack.ml.timeSeriesJob.notViewableTimeSeriesJobMessage', {
-      defaultMessage: 'Disabled because not a viewable time series job.',
+      defaultMessage: 'not a viewable time series job',
```

Comment:
I think this reads better if `it is` is added at the start `it is not a viewable time series job`

![image](https://user-images.githubusercontent.com/7405507/116542161-3fb88800-a8e4-11eb-87e0-65ce8b88d397.png)